packages/main/src/plugin/kubernetes-context-state.ts

```diff
@@ -435,9 +435,32 @@ export class ContextsManager {
     );
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  compareObjects(newu: any, oldu: any): boolean {
+    if (!newu || !oldu) return false;
```

Comment:
If could return true if both are 'zero' value

Reply:
updated the compareObject mehtod and added tests for specific update use-cases